docs/design/diagrams/managed_subscribe_module_sequence.puml

```diff
@@ -0,0 +1,65 @@
+@startuml Managed Subscribe Module
+title Managed Subscribe Module Sequence
+participant DTProvider
+participant Ibeji
+participant ManagedSubscribeModule
+participant DTConsumer
+participant Agemo
+participant Broker
+
+autonumber
+
+DTConsumer -> Ibeji: FindById (Temp)
```

Comment:
Just curious what do you mean by `FindById (Temp)` here?

Reply:
Looks like it was a holdover from an old diagram I started from, changed to entity_id

Reply:
updated